### ISSUE TITLE
Sheet headers should be visible when scrolling a sheet

### ISSUE THREAD

**Issue by Northburns** [ncannasse/castle#35]
When scrolling a sheet with many rows, the headers scroll with the rows; ie. the headers are visible only when looking at the top _screenful_ of rows.

The feature wish stated here is known as "frozen rows" in Excel and Google Sheets.

**Comment by ncannasse** [comment]
I agree, but do you have a suggestion how to do that easily with tables in CSS ?

**Comment by Northburns** [comment]
True, that'd require a bit of wonky acrobatics with just CSS.. If I come across something (maybe when CSS4 is a thing), I try to remember to return to this issue :)

Would this thing become simpler if implemented together with resizeable columns, though? (assuming that would have any weight on the feature backlog) (I'm spitballing here, of course. Sorry for not being that familiar with CastleDB's code base)

**Comment by ncannasse** [comment]
Resizable columns are a good idea but it would require to save the settings
in html5 storage on a per cdb file basis.
